astropy/table/ndarray_mixin.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+from astropy.utils.data_info import ParentDtypeInfo
+
+
+class NdarrayMixinInfo(ParentDtypeInfo):
+    _represent_as_dict_primary_data = 'data'
+
+    def _represent_as_dict(self):
+        """Represent Column as a dict that can be serialized."""
+        col = self._parent
+        out = {'data': col.view(np.ndarray)}
+        return out
+
+    def _construct_from_dict(self, map):
+        """Construct Column from ``map``."""
+        data = map.pop('data')
+        out = self._parent_cls(data, **map)
+        return out
+
+
+class NdarrayMixin(np.ndarray):
+    """
+    Mixin column class to allow storage of arbitrary numpy
+    ndarrays within a Table.  This is a subclass of numpy.ndarray
+    and has the same initialization options as ndarray().
```

Comment:
...options as *array()* (`ndarray` takes a shape and buffer...)

Reply:
Fixed.